pkg/models/configurationstruct.go

```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
```

Comment:
filename long and difficult to read. Please use underscores.

Reply:
This file should not be moved from export distro. It does not make sense to have it in models.